docs/getting-started-guides/rkt/README.md

```diff
@@ -47,8 +47,73 @@ We still have [a bunch of work](http://issue.k8s.io/8262) to do to make the expe
 
 - Note that for rkt version later than v0.7.0, `metadata service` is not required for running pods in private networks. So now rkt pods will not register the metadata service be default.
 
-- Since release [v1.2.0-alpha.5](https://github.com/kubernetes/kubernetes/releases/tag/v1.2.0-alpha.5), [rkt API service](https://github.com/coreos/rkt/blob/master/api/v1alpha/README.md)
-  is required to be running on the machine.
+- Since release [v1.2.0-alpha.5](https://github.com/kubernetes/kubernetes/releases/tag/v1.2.0-alpha.5),
+the [rkt API service](https://github.com/coreos/rkt/blob/master/api/v1alpha/README.md)
+must be running on the node.
+
+### Network Setup
+
+rkt uses the [Container Network Interface (CNI)](https://github.com/appc/cni)
+to manage container networking. By default, all pods attempt to join a network
+called `rkt.kubernetes.io`, which is currently defined [in `rkt.go`]
+(https://github.com/kubernetes/kubernetes/blob/v1.2.0-alpha.6/pkg/kubelet/rkt/rkt.go#L91).
+In order for pods to get correct IP addresses, the CNI config file must be
+edited to add this `rkt.kubernetes.io` network:
+
+#### Using flannel
+
+[flannel](https://github.com/coreos/flannel) can be configured with a
+CNI config like:
```

Comment:
@bprashanth Yes, this assumes the flanneld is a process, and should be running on the nodes before kubelet is running.

Haven't thought deep about what if we run flannel in a pod. Probably doing something like `kubelet -runonce=true` to start flannel in the host network, and then exit the kubelet.